daemon/c_fifo.c

```diff
@@ -168,12 +235,22 @@ c_fifo_loop(char *fifo_path_c0, char *fifo_path_container)
 			TRACE_ERRNO("Failed to close old reading fd");
 		}
 
+		if (!file_is_fifo(fifo_path_c0)) {
+			ERROR("Could not open FIFO at %s, stopping forwarder", fifo_path_c0);
+			exit(EXIT_SUCCESS);
```

Comment:
Is file opening failed, why return EXIT_SUCCESS?

Reply:
If the container user tampers with the fifo, imho the forwarder should just exit cleanly
I agree that "SUCCESS" sounds strange here, will replace it by exit(0)